app/views/dashboard/ng-partials/_newmovement.html.haml

```diff
@@ -1,138 +1,7 @@
-        .header-wrapper
-          %h5 Move Absolute
-      .large-12.small-12.columns
-        .content-wrapper
-          .row
-            .large-4.small-4.columns
-              %label.left.inline.shrink Gantry(X)
-            .large-4.small-4.columns
-              %input{:placeholder => "3.124m", :type => "text"}/
-            .large-4.small-4.columns
-              %button.xx-small.button.radius.gray HOME X
-          .row
-            .large-4.small-4.columns
-              %label.left.inline.shrink Cross Slide(Y)
-            .large-4.small-4.columns
-              %input{:placeholder => "1.210m", :type => "text"}/
-            .large-4.small-4.columns
-              %button.xx-small.button.radius.gray HOME Y
-          .row
-            .large-4.small-4.columns
-              %label.left.inline.shrink Tool Mount(Z)
-            .large-4.small-4.columns
-              %input{:placeholder => "0.893m", :type => "text"}/
-            .large-4.small-4.columns
-              %button.xx-small.button.radius.gray HOME Z
-          .row
-            .large-4.small-4.columns
-              %button.tiny.expand.button.radius.red STOP
-            .large-4.small-4.columns
-              %button.tiny.button.radius.green GO
-            .large-4.small-4.columns
-              %button.xx-small.button.radius.gray HOME ALL
-    .row
-      .large-12.small-12.columns
-        .header-wrapper
-          %h5 Camera 1
-      .large-12.small-12.columns
-        %img{src: "http://108.90.200.9:8080/?action=stream", style: "width: 1080px; height: auto;"}
-
-  .large-4.small-12.columns
-    .row
-      .large-12.small-12.columns
-        .header-wrapper
-          %h5 Device Status
-      .large-12.small-12.columns
-        .content-wrapper
-          .row
-            .large-6.small-6.columns
-              %label.left.inline Current Operation
-            .large-6.small-6.columns
-              %label.left.inline Idle
-          .row
-            .large-6.small-6.columns
-              %label.left.inline Next Operation
-            .large-6.small-6.columns
-              %label.left.inline Current Operation
-          .row
-            .large-6.small-6.columns
-              %label.left.inline Gantry Position(X)
-            .large-6.small-6.columns
-              %label.left.inline 3.124 m
-          .row
-            .large-6.small-6.columns
-              %label.left.inline Cross Slide Position(Y)
-            .large-6.small-6.columns
-              %label.left.inline 1.210 m
-          .row
-            .large-6.small-6.columns
-              %label.left.inline Z-Axis Position(Z)
-            .large-6.small-6.columns
-              %label.left.inline 0.893 m
-          .row
-            .large-6.small-6.columns
-              %button.tiny.expand.button.radius.red STOP
-            .large-6.small-6.columns
-              %button.tiny.button.radius.yellow RESTART
+  .large-4.medium-6.small-12.columns
+    %ng-include{:src => "'widgets/move'"}
```

Comment:
Same as above. You could just put `= render partial: "dashboard/ng-partials/widgets/move"` and get the cleanliness of partials without needing to tell Angular what's going on.